GenderPayGap.Database/Migrations/20200505085232_CreateViewsForReportsReader.cs

```diff
@@ -1,12 +1,737 @@
+                + " 		WHEN retrns.MinEmployees = 0 "
+                + " 			AND retrns.MaxEmployees = 0 "
+                + " 			THEN 'Not Provided' "
+                + " 		WHEN retrns.MinEmployees = 0 "
+                + " 			AND retrns.MaxEmployees = 249 "
+                + " 			THEN 'Less than 250' "
+                + " 		WHEN ( "
+                + " 				retrns.MinEmployees = 250 "
+                + " 				AND retrns.MaxEmployees = 499 "
+                + " 				) "
+                + " 			OR ( "
+                + " 				retrns.MinEmployees = 500 "
+                + " 				AND retrns.MaxEmployees = 999 ) OR"
+                + Environment.NewLine
+                + "                         (retrns.MinEmployees = 1000 "
+                + " 				AND retrns.MaxEmployees = 4999 "
+                + " 				) "
+                + " 			THEN CAST(retrns.MinEmployees AS VARCHAR) + ' to ' + CAST(retrns.MaxEmployees AS VARCHAR) "
+                + " 		WHEN retrns.MinEmployees = 5000 "
+                + " 			AND retrns.MaxEmployees = 19999 "
+                + " 			THEN CAST(retrns.MinEmployees AS VARCHAR) + ' to 19,999' "
+                + " 		WHEN retrns.MinEmployees = 20000 "
+                + " 			AND retrns.MaxEmployees = 2147483647 "
+                + " 			THEN '20,000 or more' "
+                + " 		ELSE 'Error min(' + CAST(retrns.MinEmployees AS VARCHAR) + '), max (' + CAST(retrns.MaxEmployees AS VARCHAR) + ')' "
+                + " 		END AS OrganisationSize "
+                + " 	,OrgPubSectType.PublicSectorDescription "
+                + " FROM dbo.Organisations AS orgs "
+                + " INNER JOIN dbo.OrganisationScopes AS OrgScopes ON OrgScopes.OrganisationId = orgs.OrganisationId "
+                + " 	AND OrgScopes.StatusId = 3 "
+                + " LEFT OUTER JOIN dbo.OrganisationSicCodeInfoView AS OrgSicCodeView ON OrgSicCodeView.OrganisationId = orgs.OrganisationId "
+                + " 	AND OrgSicCodeView.SicCodeRankWithinOrganisation = 1 "
+                + " LEFT OUTER JOIN dbo.[Returns] AS retrns ON retrns.OrganisationId = OrgScopes.OrganisationId "
+                + " 	AND retrns.AccountingDate = OrgScopes.SnapshotDate "
+                + " 	AND retrns.StatusId = 3 "
+                + " LEFT OUTER JOIN ( "
+                + " 	SELECT opst.OrganisationId "
+                + " 		,pst.Description AS PublicSectorDescription "
+                + " 	FROM dbo.OrganisationPublicSectorTypes AS opst "
+                + " 	INNER JOIN dbo.PublicSectorTypes AS pst ON pst.PublicSectorTypeId = opst.PublicSectorTypeId "
+                + " 		AND opst.Retired IS NULL "
+                + " 	) AS OrgPubSectType ON OrgPubSectType.OrganisationId = orgs.OrganisationId ");
+
+            migrationBuilder.Sql("  GRANT SELECT " + " 	ON OBJECT::dbo.[OrganisationScopeAndReturnInfoView] " + " 	TO [ReportsReaderDv]; ");
+
+            #endregion
+            
+            migrationBuilder.Sql(" ALTER VIEW [dbo].["
```

Comment:
Another `ALTER VIEW` which overrides the view created on line 220.
Similarly, try removing the block and line 220 and changing this block to be a `CREATE`